src/SmartComponents/Modals/UnassignSystemsModal.js

```diff
@@ -14,19 +14,20 @@ const UnassignSystemsModal = ({ unassignSystemsModalState = {}, setUnassignSyste
 
     const { systemsIDs, isUnassignSystemsModalOpen } = unassignSystemsModalState;
 
-    const handleModalOpen = () => {
+    const handleModalOpen = (shouldRefresh) => {
         setUnassignSystemsModalOpen({
             isUnassignSystemsModalOpen: !isUnassignSystemsModalOpen,
```

Comment:
just a minor thing: if `handleModalOpen` is called only when there is confirm or close button clicked, then it is better to pass `false` explicitly to the `isUnassignSystemsModalOpen` flag:

```suggestion
            isUnassignSystemsModalOpen: false,
```